components/LinkCustom/LinkCustom.test.tsx

```diff
@@ -12,4 +12,4 @@ describe('LinkCustom Component', () => {
     expect(link).toHaveTextContent('Test link');
     expect(link).toHaveAttribute('href', 'https://example.com');
   });
-});
+});
```

Comment:
NIT: How come there's an extra line here?

Reply:
I didn't want the first extra line to feel lonely ... both gone now.

Reply:
I'm not seeing this one in the file